### ISSUE TITLE
Ubuntu 15.10 still has restore issues

### ISSUE THREAD

**Issue by awishformore** [gitterHQ/desktop#121]
The tray icon is working fine on my new Ubuntu 15.10 installation now, after installing libappindicator1.

Unfortunately, I can't seem to open the Gitter window. When I start Gitter, I get the tray icon (with correct recents) and a Launcher icon. However, I can't seem to restore the Gitter window by clicking either the options on the tray icon or the Launcher icon.

If I select "Sign Out" on the tray icon, I will get the login screen. After logging in, the window disappears again.

I I select "Quit" on the Launcher icon, the tray icon stays. Activating one of the recents options will flash a window for a fraction of a second, before it disappears again.

In short, I have no way to actually use the desktop client at this point.

**Comment by MadLittleMods** [comment]
Hey @awishformore,

I will get a Ubuntu 15.10 up and running but the restore functionality appears to work just fine in Ubuntu 14.04.03.

**Comment by awishformore** [comment]
It works for me on my PC, but not on my laptop. Is there a "start minimized" feature somewhere? I vaguely remember turning something like that on on the laptop. If that's indeed the case, that's where the issue could stem from.

**Comment by MadLittleMods** [comment]
@awishformore There isn't a start minimized option: https://github.com/gitterHQ/desktop/issues/107

You can try using https://github.com/gitterHQ/desktop/issues/69#issuecomment-126656798 to get your  state in order.

**Comment by nicolas33** [comment]
@awishformore For whatever reason, I had to double click the icon to get the window at some point in time (not currently). Might worth a try!
